package.json

```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "scripts": {
     "test": "jest",
-    "sass": "sass --watch src/styles:build",
+    "sass": "sass --watch src/styles:src/build",
```

Comment:
Not related to this spot specifically, but this is a _very_ large commit. In the future, I'd try to organize individual commits around ideas. As an example, maybe have a single commit for changing this script, one for some of the other CSS changes, etc.